during_workshop.md

```diff
@@ -193,11 +161,14 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
         f"Processing took {str(processingTime)} seconds. Translation is: {subtitle}",
         status_code=200
     )
-
 ```
 
 Once you've updated the code, you can run `func start` again to run the new version.
 
+> If you'd like to debug the Azure Function App locally you'll need to open the Function App subfolder (`AcmeSubProject`) in VSCode (using File => Open Folder) and accept the following prompt: 
+> ![Azure Function Init Prompt](./images/Azure%20Functions%20Setup.png)
+> This should generate a `launch.json` file so that you can debug the application using the Run & Debug button on the Activity Bar

```

Comment:
I'm slightly surprised we can't just put a placeholder AcmeSubProject folder with those files in with maybe some extra venv step here.. but then to be honest I guess it was easy for me to get this working following your instructions here

On reflection, ignore this. The instructions worked perfectly 🤷